ignite/contrib/handlers/time_profilers.py

```diff
@@ -437,3 +438,321 @@ def odict_to_str(d: Mapping):
+                if not self._is_internal_handler(func):
+                    engine._event_handlers[e][i] = (self._create_wrapped_handler(func, e), args, kwargs)
+
+        # processing timer
+        engine.add_event_handler(Events.ITERATION_STARTED, self._processing_timer.reset)
+        engine._event_handlers[Events.ITERATION_COMPLETED].insert(0, (self._timeit_processing, (), {}))
+
+        # dataflow timer
+        engine.add_event_handler(Events.GET_BATCH_STARTED, self._dataflow_timer.reset)
+        engine._event_handlers[Events.GET_BATCH_COMPLETED].insert(0, (self._timeit_dataflow, (), {}))
+
+        # revert back the wrapped handlers with original handlers at the end
+        engine.add_event_handler(Events.COMPLETED, self._detach_profiler_handlers)
+
+    def attach(self, engine: Engine) -> None:
+        if not isinstance(engine, Engine):
+            raise TypeError("Argument engine should be ignite.engine.Engine, but given {}".format(type(engine)))
+
+        if not engine.has_event_handler(self._as_first_started):
+            engine._event_handlers[Events.STARTED].insert(0, (self._as_first_started, (engine,), {}))
+
+    def get_results(self) -> List[List[Union[str, float]]]:
+        """
+        Method to fetch the aggregated profiler results after the engine is run
+
+        .. code-block:: python
+
+            results = profiler.get_results()
+
+        """
+        total_eh_time = sum(
+            [
+                sum(self.event_handlers_times[e][h])
+                for e in self.event_handlers_times
+                for h in self.event_handlers_times[e]
+            ]
+        )
+        total_eh_time = round(float(total_eh_time), 5,)
+
+        def compute_basic_stats(data: Sequence) -> List[Union[str, float]]:
+            # compute on non-zero data:
+            data = data[data > 0]
+            total = round(torch.sum(data).item(), 5) if len(data) > 0 else "not triggered"
+            min_index = ("None", "None")
+            max_index = ("None", "None")
+            mean = "None"
+            std = "None"
+            if len(data) > 0:
+                min_index = (round(torch.min(data).item(), 5), torch.argmin(data).item())
+                max_index = (round(torch.max(data).item(), 5), torch.argmax(data).item())
+                mean = round(torch.mean(data).item(), 5)
+                if len(data) > 1:
+                    std = round(torch.std(data).item(), 5)
+            return [total, min_index, max_index, mean, std]
+
+        event_handler_stats = [
+            [h, getattr(e, "name", str(e)), *compute_basic_stats(torch.FloatTensor(self.event_handlers_times[e][h]))]
+            for e in self.event_handlers_times
+            for h in self.event_handlers_times[e]
+        ]
+        event_handler_stats.append(["Total", "", total_eh_time, "", "", "", ""])
+        event_handler_stats.append(
+            ["Processing", "None", *compute_basic_stats(torch.FloatTensor(self.processing_times))]
```

Comment:
Let's keep the type of `compute_basic_stats` input as sequence and if we need to create a float tensor, we do it inside the function using `torch.tensor(.., dtype=torch.float32)`